loopy/kernel/creation.py

```diff
@@ -2382,11 +2383,6 @@ def make_kernel(domains, instructions, kernel_data=["..."], **kwargs):
 
 
 def make_function(*args, **kwargs):
-    lang_version = kwargs.pop('lang_version', None)
-    if lang_version:
-        raise LoopyError("lang_version should be set for program, not "
-                "functions.")
```

Comment:
I already don't remember what we decided on how language versions should work in a translation unit. Could you remind me (and add docs)?

Reply:
We had agreed on letting `lang_version`/`options` be broken for a bit until we have managed to merge in kcv3 into master, and then separate out the kernel options and translation unit options.